airflow/api_connexion/endpoints/dataset_endpoint.py

```diff
@@ -70,7 +70,7 @@ def get_datasets(
     total_entries = session.query(func.count(Dataset.id)).scalar()
     query = session.query(Dataset)
     if uri_pattern:
-        query = query.filter(Dataset.uri.ilike(f"%{uri_pattern}%"))
+        query = query.filter(Dataset.uri.ilike(uri_pattern))
```

Comment:
Could we add a description in the API yaml docs to tell users to add `%` for less strict equality lookups?

Reply:
i added some description for the API... please take a look